source/sr/1.0.0/index.html.haml

```diff
@@ -0,0 +1,311 @@
+---
+description: Vodite changelog
+title: Vodite changelog
+language: hr
```

Comment:
Oh there was a mistake here @zoranzoki21. I have to fix that otherwise the wrong translation name will appear in the <select> element on the site.